doc/source/data/examples/BUILD

```diff
@@ -5,3 +5,11 @@ filegroup(
     srcs = glob(["*.ipynb"]),
     visibility = ["//doc:__subpackages__"]
 )
+
+py_test_run_all_notebooks(
```

Comment:
@amogkam The inference actors will OOM in CI. the batch_size is already very small (4). I think we should probably move it to the release tests instead.

Reply:
By using GPUs by default, the test passes now.